docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +61,162 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
 NAME                                                                             CLUSTER                   REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE     VERSION
 machinedeployment.cluster.x-k8s.io/clusterclass-quickstart-linux-workers-XXXX    clusterclass-quickstart   1          1       1         0             Running   7m29s   v1.22.0
 ```
+
+## Scale a MachineDeployment
+When using a managed topology scaling of MachineDeployments, both up and down, should be done through the Cluster topology.
```

Comment:
```suggestion
When using a managed topology scaling of MachineDeployments, both up and down, is done through the Cluster topology.
```
Maybe, to make the statement a bit stronger. Scaling on the MachineDeployment will only work if `replicas` is not set in the topology.

In case we want to make that case visible here I think it would be good to make that more explicit. (maybe as a separate note with aside)

Reply:
'Should' here isn't used in a conditional sense, but in an imperative sense. It's trying to tell the user what to do, not describing how the system works. If you feel strongly about it I can change it, but 'is' doesn't come across as an instruction here.

Reply:
:+1: